mailscanner/languages/de.php

```diff
@@ -32,199 +32,205 @@
-    'wl07' => 'Weisse Liste',
-    'bl07' => 'Schwarze Liste',
-    'reset07' => 'Zurücksetzen',
-    'add07' => 'Hinzufügen',
-    'delete07' => 'Löschen',
-
-    // 08-quarantine.php
-    'folder08' => 'Ordner',
-    'folder_0208' => 'Ordner',
-    'items08' => 'Einträge',
-
-    // 09-filter.inc
-    'activefilters09' => 'Aktive Filter',
-    'none09' => 'Keiner',
-    'addfilter09' => 'Filter hinzufügen',
-    'loadsavef09' => 'Filter öffnen/speichern',
-    'tosetdate09' => 'Für das Datum muss das Format JJJJ-mm-tt verwendet werden',
-
-    // Added in 2015-06-23
-    // 04-details.php
-    'quarantine04' => 'Quarantäne',
-
-    // 03-functons.php
-    'score03' => 'Bewertung',
-    'matrule03' => 'Zutreffende Regel',
-    'description03' => 'Beschreibung',
-
-    // 10-other.php
-    'tools10' => 'Werkzeuge',
-    'usermgnt10' => 'Benutzerverwaltung',
-    'viewconfms10' => 'MailScanner Einstellungen betrachten',
-    'editmsrules10' => 'MailScanner Regeln bearbeiten',
-    'updatesadesc10' => 'Aktualisiere die Beschreibungen der SpamAssassin Regeln',
-    'updatemcpdesc10' => 'Aktualisiere die Beschreibungen der MCP Regeln',
-    'updategeoip10' => 'Aktualisiere die GeoIP Datenbank',
-
-    // 11-sf_versions.php
-    'softver11' => 'Softwareversionen',
-    'nodbdown11' => 'Keine Datenbank heruntergeladen',
-    'version11' => 'Version',
-
-    // 12-user_manager.php
-    'usermgnt12' => 'Benutzerverwaltung',
-    'username12' => 'Benutzername',
-    'fullname12' => 'Voller Name',
-    'type12' => 'Typ',
-    'spamcheck12' => 'Spam Prüfung',
-    'spamscore12' => 'Spam Bewertung',
-    'spamhscore12' => 'Hohe Spam Bewertung',
-    'action12' => 'Aktionen',
-    'edit12' => 'Bearbeiten',
-    'delete12' => 'Löschen',
-    'filters12' => 'Filter',
-    'newuser12' => 'Neuer Benutzer',
-    'forallusers12' => 'Für alle Benutzer ausser dem Administrator muss der Benutzername eine E-Mail Adresse sein',
-    'username0212' => 'Benutzername:',
-    'name12' => 'Name:',
-    'password12' => 'Passwort:',
-    'usertype12' => 'Benutzertyp:',
-    'user12' => 'Benutzer',
-    'domainadmin12' => 'Domänen Administrator',
-    'admin12' => 'Administrator',
-    'quarrep12' => 'Quarantäne Bericht:',
-    'senddaily12' => 'Täglichen Bericht senden?:',
-    'quarreprec12' => 'Empfänger des Quarantäne Berichts:',
-    'overrec12' => 'Empfangadresse für Quarantäne Bericht überschreiben?<BR>(falls leer, wird der Benutzername verwendet)',
-    'scanforspam12' => 'Auf Spam prüfen:',
-    'scaneforspam12' => 'E-Mail auf Spam prüfen?',
-    'pontspam12' => 'Spam Bewertung:',
-    'hpontspam12' => 'Hohe Spam Bewertung:',
-    'usedefault12' => 'Voreinstellungen benutzen',
-    'action_0212' => 'Aktion:',
-    'reset12' => 'Zurücksetzen',
-    'areusuredel12' => 'Soll der Benutzer tatsächlich gelöscht werden',
-    'errorpass12' => 'Passwörter stimmen nicht überein',
-
-    'i18_missing' => 'Keine deutsche Übersetzung',
+	// 01-login.php
```

Comment:
please don't replace the spaces with tabs, I would like to keep PSR compatibility (even with all the limits that the actual code has)